### ISSUE TITLE
Why is the dropdown only triggered after a space?

### ISSUE THREAD

**Issue by ooker777** [dsarman/better-inline-fields#2]
For notes whose filenames have multiple words, then only after typing the space in between the words that the dropdown is triggered. I assume that it's for less clutter when you are typing. Is that correct? The issue is that if I want to create a new note, then it will make the filename with a space, e.g. `multiple words .md`. This creates a little annoying, since Obsidian reads `[[multiple words ]]` (with a space) the same with `[[multiple words]]` and links to `multiple words.md`, while the plugin will create `multiple words .md`.

**Comment by dsarman** [comment]
I am not sure if I understand your problem. The only space that should be required is the one after the inline field name and double colon `fieldName::⠀`. ~~I can change it so it is not required in the next release~~ This behavior is already changed in release 0.1.1 so that the starting space is not required. If that is not what you meant, please provide a video or gif recording of what leads to the problem you are seeing.

**Comment by ooker777** [comment]
No it's not. Here is the screen recorder. You can see that there are two new created files: `newfile.md` and `newfile .md`. They are not the same. I hope it's clearer
![](https://i.imgur.com/BuahpMG.gif)

**Comment by dsarman** [comment]
This is strange and should not happen. I can see the `+ Create New` option the whole time. I tried to reproduce the issue on my end but I was not successfull. Could try to open your [devtools console window](https://developer.chrome.com/docs/devtools/shortcuts/#open) and see if you can see any errors pop up when doing the same as on the recording above?

**Comment by ooker777** [comment]
It turns out that it conflicts with the [Various Complements](https://tadashi-aikawa.github.io/docs-obsidian-various-complements-plugin/ "Home - Various Complements") plugin. I check the console and find nothing. Do you have a debug mode?

**Comment by dsarman** [comment]
I installed the Various Complements plugin and tried to reproduce the issue, but I was still not successfull. The problem may depend on the plugins setting and/or Obsidian vault content. Maybe you could try disabling the "Complement automatically" setting of the Various Complements plugin, I have a hunch that that could help.

**Comment by ooker777** [comment]
Yes that's the cause of the issue. Thanks. I guess you cannot have two suggesters at the same time

**Comment by dsarman** [comment]
I am closing this as won't fix and I will add an information about the incompatibility to the README.
